README.md

```diff
@@ -166,6 +193,40 @@ language, _ = sdk.language.get(request_id = language_jobs[0].id)
 
 Note that the above accepts a list of ```fild_ids```
 
+# Date Normalization
+DocAI can be used to normalize strings that contain dates, so that the `year`, `month` and `day` are returned.
+
+```python
+from zdai import ZDAISDK
+import json
+
+with open('quick_demo.json', 'r') as file:
+    data = json.loads(file.read())
+    phrases = data.get('normalization')
+    mlc_files = data.get('mlc_files')
```

Comment:
Looks like some unnecessary code snuck in here.
```suggestion
```

Reply:
5f656f1e53c5db6fdba711a10b14bc92a3e4b71e